index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
- [ ] It is required to give your `h1` element a class name that would be use to style your header in your css file. Kindly add class name to your `h1` and use the class name in your css file for styling.